tests/cloudformation/checks/resource/aws/example_ALBDropHttpHeaders/FAIL.yaml

```diff
@@ -0,0 +1,42 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Resources:
+  FailDefaultType:
+    Type: AWS::ElasticLoadBalancingV2::LoadBalancer
+    Properties:
+      Name: test
+      Subnets: 
+        - test-0
+        - test-1
+  FailExplicitALB:
+    Type: AWS::ElasticLoadBalancingV2::LoadBalancer
+    Properties:
+      Name: test
+      Subnets: 
+        - test-0
+        - test-1
+      Type: application
+      LoadBalancerAttributes:
+        - Key: routing.http.drop_invalid_header_fields.enabled
+          Value: "false"
+  FailExplicitFalse:
+    Type: AWS::ElasticLoadBalancingV2::LoadBalancer
+    Properties:
+      Name: test
+      Subnets: 
+        - test-0
+        - test-1
+      Type: application
+      LoadBalancerAttributes:
+        - Key: routing.http.drop_invalid_header_fields.enabled
+          Value: "false"
```

Comment:
There is no difference between both resources, I think you wanted to the set `Type: application` without defining `LoadBalancerAttributes`, right?